controller-server/src/test/java/com/yahoo/vespa/hosted/controller/ControllerTest.java

```diff
@@ -1324,6 +1324,33 @@ void testCloudAccount() {
         }
     }
 
+    @Test
+    void testCloudAccountWithDefaultOverride() {
+        var context = tester.newDeploymentContext();
+        var prodZone1 = productionUsEast3.zone();
+        var prodZone2 = productionUsWest1.zone();
+        var cloudAccount = "012345678912";
+        var application = new ApplicationPackageBuilder()
+                .cloudAccount(cloudAccount)
+                .region(prodZone1.region())
+                .region(prodZone2.region(), "default")
+                .build();
+
+        // I don't know why this makes the test pass :(
```

Comment:
@mpolden Please explain and I'll remove the comment 😄

Reply:
The `012345678912` account is used for `prodZone1` and access to do this must be granted through the feature flag.